docs/shared/packages/vue/vue-plugin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Overview of the Nx Vue Plugin
+description: The Nx Plugin for Vue contains generators for managing Vue applications and libraries within an Nx workspace. This page also explains how to configure Vue on your Nx workspace.
+---
+
+{% callout type="caution" title="`@nx/vue` is not available yet" %}
+The `@nx/vue` plugin is not available yet. It will be released soon.
+{% /callout %}
```

Comment:
We don't need this callout, since the package doesn't show up until we mark it as `private: false`